tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -81,6 +82,18 @@ if (tensorflow_OPTIMIZE_FOR_NATIVE_ARCH)
   endif()
 endif()
 
+# MSVC SIMD instructions
```

Comment:
Do you rather add it in an `else()` in the same closure of the `arch=native` or leave it separate like it is?